bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -4326,6 +4331,25 @@ String addSourceToArtifactAttribute(BlackboardAttribute attr, String source) thr
 	 *                          within tsk core
 	 */
 	public BlackboardArtifact.Type addBlackboardArtifactType(String artifactTypeName, String displayName) throws TskCoreException, TskDataException {
+		
+		return addBlackboardArtifactType(artifactTypeName, displayName, BlackboardArtifact.Category.EXTRACTED_DATA);
+	}
+
+	/**
+	 * Add an artifact type with the given name and category. Will return an artifact Type.
+	 *
+	 * @param artifactTypeName System (unique) name of artifact
+	 * @param displayName      Display (non-unique) name of artifact
+	 * @param category		   Artifact type category.
+	 * 
+	 *
+	 * @return Type of the artifact added.
+	 *
+	 * @throws TskCoreException exception thrown if a critical error occurs
+	 * @throws TskDataException exception thrown if given data is already in db
+	 *                          within tsk core
+	 */
+	public BlackboardArtifact.Type addBlackboardArtifactType(String artifactTypeName, String displayName, BlackboardArtifact.Category category) throws TskCoreException, TskDataException {
```

Comment:
We should not add this as public API to SleuthkitCase. I have a task from Brian to do after this effort that entails "migrating" all public blackboard related methods from SleuthkitCase to Blackboard. The "migration" will not be a refactoring per se, because we will be deprecating the SleuthkitCase blackboard methods. Either putting this entirely in Blackboard or at least making this a package method instead of a public method will mean there will one less thing to deprecate.